README.md

```diff
@@ -39,60 +27,42 @@
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
+### Tech Stack <a name="tech-stack"></a>
+▶ Postgresql
 
-- **[Use Ruby Basics]**
+▶ Database
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 💻 Getting Started <a name="getting-started"></a>
 
-<!-- GETTING STARTED -->
+This repository includes files with plain PostgreSQL that can be used to recreate a database:
 
-## 💻 Getting Started <a name="getting-started"></a>
+- Use [schema.sql](./database/schema.sql) to create all tables
+- Find the visual representatio of the database inside the folder : <img src='./database/catalog_of_my_things.png' />
 
-To get a local copy up and running, follow these steps.
+<a name="readme-top"></a>
 
-- Clone the [project using github](https://github.com/Zilola-Nazarova/ruby-capstone-project.git)
+### Prerequisites
+In order to run this project, you must have installed and configured the following tools in your local environment:
 
+1). Ruby
+  - the [Ruby](https://www.ruby-lang.org/en/ownload/) should be installed in your machine.
+  - Run `ruby main.rb`.
 
+2.) PostgreSQL
 
 ### Setup
 
-After cloning this repository to your desired folder:
+Clone this repository to your desired folder:
 
 ```sh
 
   cd my-folder
+  git clone: https://github.com/Zilola-Nazarova/ruby-capstone-project.git
 
 ```
 
```

Comment:
Nice job on providing detailed `Setup` instructions.

I was wondering if we could add also the remaining commands:


### Install

Install the dependencies:

```sh
gem install
```

### Usage

To run the project, execute the following command:

```sh
ruby main.rb
```

### Run tests

To run tests, run the following command:

```sh
rspec
```

You can also check linter errors:

```sh
rubocop
```